Jenkinsfile.TEST

```diff
@@ -0,0 +1,102 @@
+pipeline {
```

Comment:
Using a name different than `Jenkinsfile` allow me to configure a _GitHub Organization Job_:

![image](https://user-images.githubusercontent.com/1157864/114250448-1dca8600-9974-11eb-86e8-d7f89f76393e.png)

It automatically scans all [FromDoppler organization](https://github.com/FromDoppler)'s repositories without needed to configure them manually.

![image](https://user-images.githubusercontent.com/1157864/114250509-4f435180-9974-11eb-9a69-7a9823d3e8df.png)

I do not want to use `Jenkinsfile` as the filename because there are already existing repositories with legacy multibranch pipeline workflows.

![image](https://user-images.githubusercontent.com/1157864/114250524-5a967d00-9974-11eb-9f65-977ce64263e2.png)

But `Jenkinsfile.TEST` is not a good name, what do you think about `Jenkinsfile.Doppler`, `Jenkinsfile.FromDoppler`, `doppler-ci.groovy`?

Reply:
I like Jenkinsfile.FromDoppler

Reply:
I am trying with a symlink, if it works we will get the best of both worlds: 
* Having the standard filename
* Allow us to filter the projects